react-client/src/routes/CreateTutorial.js

```diff
@@ -1,21 +1,26 @@
 import React, { Component } from "react"
 import Section from "../components/createTutorial/section"
 
+//Bootstrap
+import { Button } from "react-bootstrap"
+
 const sectionsDataModel = {
-  sections: [
+  sectionName: String,
+  sectionDetails: [
```

Comment:
I'm super curious, what is this called? It's effectively the equivalent of an interface. DM it me please hehe.

Reply:
It's just an empty object. Like a model. So every time you want to add in a section or something, pushing this Object into the array, will effectively allow us to map render onto the screen, and also collect/store data. I'll explain further if you want